cmd/podman/spec.go

```diff
@@ -685,3 +696,30 @@ func (c *createConfig) CreatePortBindings() ([]ocicni.PortMapping, error) {
 	}
 	return portBindings, nil
 }
+
+//AddPrivilegedDevices iterates through host devices and adds them to the spec
```

Comment:
Nit: can you adjust the comment to say it adds all host devices to the spec?

Reply:
sure.